CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * gh-runner: support arm64 target architecture
 * build: fix `sparc-phoenix` toolchain linking spec
 * build: make `sparc-phoenix` multilib
+* devel: add tini 19.0-1 to amd64 image
```

Comment:
don't modify the changelog of already-released image.

Please put it under `NEXT VERSION` for now (these entries will be used when new image would be released)